src/resources/postcss/base/forms/_text.pcss

```diff
@@ -24,27 +24,44 @@
 	}
 
 	.tribe-common-form-control-text__input {
-		@mixin desktop-body-2-bold;
-
-		border: 1px solid var(--color-border-default);
-		padding: 12px 18px;
+		@mixin desktop-body-1;
+		border: none;
+		border-bottom: 1px solid var(--color-border-default);
+		padding: 12px 20px 12px 0;
```

Comment:
let's use `--spacer` variables for these?

Reply:
we also have new spacer variables, so be sure to pull the latest `tribe-common-styles`.